jedi/api/environment.py

```diff
@@ -246,23 +242,6 @@ def find_system_environments():
             pass
```

Comment:
_get_version was deleted in the super class, so we can remove it here safely as well. However SameEnvironment shouldn't execute anything (It's mostly there for REPL completions).

I think we should probably be doing something like this:

```
class Environment(object):
    def __init__(self, executable):
      self.executable = executable
      self._load_attributes()

   def _load_attributes(self):
      ... # subprocess stuff

class SameEnvironment():
   def __init__(self):
      super(SameEnvironment, self).__init__(sys.executable)

   def _load_attributes(self):
      self.version_info = sys.version_info
      self.path = sys.prefix
```

Reply:
I think it can be simpler - pushed 2bb32e5.